src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -118,6 +118,7 @@ class DDPMScheduler(SchedulerMixin, ConfigMixin):
 
     _compatibles = [e.name for e in KarrasDiffusionSchedulers]
     order = 1
+    _is_ode_scheduler = False
```

Comment:
Could we create a new scheduler class here, `DDPMParallelScheduler`?

I'd like to keep the DDPM scheduler simple if possible :-)

Reply:
ok done!